sigle/src/config.ts

```diff
@@ -2,6 +2,10 @@ export const sigleConfig = {
   env: process.env.NODE_ENV,
   isServer: typeof window === 'undefined',
   appUrl: process.env.APP_URL as string,
+  baseUrl:
```

Comment:
let's inject the values from the .env files :)

Reply:
hmm seems like the value is coming up as undefined when I try to pull them from .env.local - do you know what could be potentially causing them to show up as undefined?

Reply:
what's the name of the variable?

Reply:
You might want to have a look at sigle/next.config.js to inject them :)

Reply:
yep that did it, thanks :)